hi-ml/src/health_ml/utils/data_augmentations.py

```diff
@@ -37,45 +37,44 @@ def __init__(self, theta: float = 0.) -> None:
         self.hed_from_rgb = torch.tensor([[1.87798274, -1.00767869, -0.55611582],
                                           [-0.06590806, 1.13473037, -0.1355218],
                                           [-0.60190736, -0.48041419, 1.57358807]])
+        self.log_adjust = torch.log(torch.tensor(1E-6))
 
-    @staticmethod
-    def adjust_hed(img: torch.Tensor,
-                   theta: float,
-                   stain_from_rgb_mat: torch.Tensor,
-                   rgb_from_stain_mat: torch.Tensor
-                   ) -> torch.Tensor:
+    def adjust_hed(self, img: torch.Tensor) -> torch.Tensor:
```

Comment:
[Minor] Could it not be useful to keep this as a static? Maybe turning `rgb_from_hed`, `hed_from_rgb`, and `log_adjust` into class attributes to limit the number of arguments?